changelogs/fragments/3478-yaml-callback.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - "yaml callback plugin - avoid modifying PyYAML so that other plugins using it on the controller, like the ``to_yaml`` filter, produce different output (https://github.com/ansible-collections/community.general/issues/3471, https://github.com/ansible-collections/community.general/pull/3478)."
```

Comment:
nit: should this say "don't* produce different output"?

Reply:
```suggestion
  - "yaml callback plugin - avoid modifying PyYAML so that other plugins using it on the controller, like the ``to_yaml`` filter, do not produce different output (https://github.com/ansible-collections/community.general/issues/3471, https://github.com/ansible-collections/community.general/pull/3478)."
```